src/common/util/common.ts

```diff
@@ -65,3 +65,18 @@ export function animate(fns: Array<() => void>, delay: number): void {
     setTimeout(() => animate(rest, delay), delay);
   }
 }
+
+// Decode a Firebase array key into a timestamp (https://stackoverflow.com/a/27910048)
+export function decodeFirebaseKey(key: string): number {
```

Comment:
Should we start adding timestamps to game entries in firebase so that we won't have to do this crazy hack for future games?

Reply:
Let's do that in a separate PR